packages/eds-lab-react/src/components/SideBar/SidebarLink/index.tsx

```diff
@@ -35,15 +28,10 @@ const {
 } = tokens
 
 type ContainerProps = {
-  active?: boolean
-} & StrippedButton
-
-type StrippedButton = Omit<
-  ButtonProps,
-  keyof ButtonHTMLAttributes<HTMLButtonElement>
->
+  $active?: boolean
```

Comment:
why did you choose `$active` as prop name with $?

Reply:
I tested again with just `active` after the other change and nope, still got the warning, but with `$active` no warning so it seems to be needed here